src/cfnlint/custom_rules.py

```diff
@@ -25,16 +26,18 @@ def check(filename, template):
             if rule.valid and rule.resourceType[0] != '#':
                 try:
                     resource_properties = template.get_resource_properties([rule.resourceType])
-                    result = Operator[rule.operator](rule, resource_properties)
-                    if result != rule.operator:
-                        matches.append(cfnlint.rules.Match(line_number, '0', '0', '0', filename, CustomRule('E9999'), result, None))
+                    result = Operator[rule.operator](template, rule, resource_properties)
```

Comment:
`result` is very generic, is this a match? a list of matches?

Reply:
Will refactor to something along the lines of operator_results - I'm trying to avoid using the word matches as the final returned results are named matches

Reply:
Updated in new patch